IsraelHiking.Web/src/application/services/route-statistics.service.spec.ts

```diff
@@ -195,25 +195,25 @@ describe("RouteStatisticsService", () => {
 
         let statistics = service.getStatistics(routeData, null, null, null, false);
 
-        expect(statistics.gain).toBeCloseTo(59.9,1);
-        expect(statistics.loss).toBeCloseTo(-59.8,1);
+        expect(statistics.gain).toBeCloseTo(278.4,1);
+        expect(statistics.loss).toBeCloseTo(-238.4,1);
         expect(statistics.length).not.toBe(0);
     });
 
     it("Should get statistics on part of route", () => {
         let routeData = {
             segments: [
                 {
-                    latlngs: [{ lat: 0, lng: 0, alt: 0 }, { lat: 0, lng: 0.01, alt: 2 }, { lat: 0, lng: 0.02, alt: 1 }]
+                    latlngs: [{ lat: 0, lng: 0, alt: 0 }, { lat: 0, lng: 0.01, alt: 200 }, { lat: 0, lng: 0.02, alt: 100 }]
                 }
             ]
         } as RouteData;
         let statistics = service.getStatistics(routeData, null, null, null, false);
-        let start = service.interpolateStatistics(statistics, 0.5);
-        let end = service.interpolateStatistics(statistics, 1);
+        let start = service.interpolateStatistics(statistics, 100);
```

Comment:
x was expected in Km I think (related to the `/`1000` change I guess, I should probably document this method better in terms of what it receives...
So passing 100 and 200 seems strange...